jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/LifeCycleBaseTest.java

```diff
@@ -2388,7 +2390,9 @@ public void beforeTaskStartedEvent(TaskEvent event) {
 
             @Override
             public void beforeTaskCompletedEvent(TaskEvent event) {
-                assertNull(event.getTask().getTaskData().getTaskInputVariables());                
+                assertNotNull(event.getTask().getTaskData().getTaskInputVariables());
+                assertEquals(1, event.getTask().getTaskData().getTaskInputVariables().size());
+                assertTrue(event.getTask().getTaskData().getTaskInputVariables().containsKey("input"));                
```

Comment:
Why is it now not null? Because of context.loadTaskVariables(task); in CompleteTaskCommand? Isn't it a change in behaviour so a doc update is needed? Same for afterTaskCompletedEvent.